Kickstarter-iOS/Views/Cells/DiscoveryPostcardCell.swift

```diff
@@ -189,6 +189,7 @@ internal final class DiscoveryPostcardCell: UITableViewCell, ValueCell {
     self.viewModel.outputs.metadataData
       .observeForUI()
       .observeValues { [weak self] data in
+        self?.metadataIconImageView.isHidden = data.iconImage == nil
```

Comment:
I guess by "easer to test and understand", I mean that a unique output tells us exactly what we're expecting to update on the UI specifically. That's why we typically don't have one output to update the entire view because that doesn't tell us what we're doing with individual UI components.